src/frontend/Docs/Entry.elm

```diff
@@ -191,6 +269,19 @@ longFunctionAnnotation nameDict args result =
     List.map2 (++) starters tipeHtml
 
 
+longFunctionAnnotationSearch : String -> Name.Dictionary -> List Type -> Type -> List (List Html)
+longFunctionAnnotationSearch basePath nameDict args result =
+  let
+    tipeHtml =
+      List.map (Type.toHtmlWithBasePath basePath nameDict Type.Func) (args ++ [result])
+
+    starters =
+      [ text "    ", colon, text "  " ]
+      ::
```

Comment:
That is the same as in `longFunctionAnnotation`, which is existing code. As said in the PR, this needs to get DRYed up. I'm just not sure if I want to add a `basePath` or `Page.Context` to the existing "normal" views.